themes/helm/layouts/_default/section.html

```diff
@@ -1,6 +1,7 @@
 {{ partial "header.html" . }}
 
-  <div class="container container-full">
+  <div class="container container-full 
+  {{ if in .Page.RelPermalink "docs" }} docs {{ else }} not-docs {{ end }} ">
```

Comment:
A better way to achieve this:

```
{{ if eq .Section "docs: }}
``

Reply:
Thanks @lucperkins, this solves my hugo issue! 💯